Description: cogl-defines from gl breaks cross gles2 builds;
 needs to guard the GL/gl.h include if gl2.h from gles2 was already included
Bug-Ubuntu: https://launchpad.net/bugs/623376
Author: Alexander Sack <asac@linaro.org>
Forwarded: ?

Index: cogl-1.7.4/cogl/cogl-defines.h.in
===================================================================
--- cogl-1.7.4.orig/cogl/cogl-defines.h.in
+++ cogl-1.7.4/cogl/cogl-defines.h.in
@@ -29,7 +29,11 @@
 #define __COGL_DEFINES_H__
 
 #include <glib.h>
+#ifndef __gl_h_
+#ifndef __gl2_h_
 @COGL_GL_HEADER_INCLUDES@
+#endif
+#endif
 
 G_BEGIN_DECLS
 
